lib/assessment.md

```diff
@@ -41,7 +56,20 @@ What selectors could we write to reference each of the following categories of e
 * Any `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
-
+```CSS
+.big{
+    background-color: yellow;
+}
+#contentPane{
+    background-color: blue;
+}
+.important.red{
+    background-color: red;
+}
+div, p{
+    background-color: orange;
+}
```

Comment:
No, this is not correct. This would grab _all_ `div`s and _all_ `p`s. To grab only those `p` tags whose parent element is a `div`, the selector is `div > p`.